### PR TITLE
First feature

### PR DESCRIPTION
**Pull request by Usagib**
OOP Milestone#1 Pull request

### CODE REVIEW

**Comment by Usagib** [comment]
Hello team just finished the project setup milestone, waiting for code review, 
happy coding awesome day!

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great start team :clap: :clap: Unfortunately, you need to make some changes in order to get approval for this milestone. check :arrow_down: 

- **[REQUIRED]** It has `lib` directory (with .gitkeep file OR any other **empty** file(s)). ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/lib_dir_exist.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- **[REQUIRED]** The project should have only one **main.rb** please remove the other one.

- **[OPTIONAL]** Add more info to your readme file. It doesn't have to contain a sophisticated description but at least a brief intro.

### That's it :relaxed: When finished, kindly resubmit another code review request :pray: 

Happy coding :v: 
**_Ahmed Mahfoudh_**

**Review by suretrust — COMMENTED** [review]
Hi team!
### Amazing job! :raised_hands: Let's get this merged! :+1: 

_Good luck with the next milestones and happy coding!_ :tada:

**Pull request merged**
